.gitignore

```diff
@@ -82,3 +82,6 @@ tags
 .kube/*
 /build
 .qz/*
+
+### metric ###
+framework/pkg/metric
```

Comment:
ignored?

Reply:
Have some temporary files here. I will detele them and change back ignored